AdventureWorks_medium_questions.sql

```diff
@@ -0,0 +1,78 @@
+-- AdventureWorks Medium Questions
+-- SOURCE :  https://sqlzoo.net/wiki/AdventureWorks_medium_questions
+
+--QUESTION 1: A "Single Item Order" is a customer order where only one item is ordered. Show the SalesOrderID and the UnitPrice for every Single Item Order.
+
+select SalesOrderID, UnitPrice
+from SalesOrderDetail
+where OrderQty = 1
+
+
+-- QUESTION 2: Where did the racing socks go? List the product name and the CompanyName for all Customers who ordered ProductModel 'Racing Socks
+
+select a.Name,d.CompanyName
+from ProductModel as p
+
+inner join Product   as a
```

Comment:
The aliases of the tables could be more understandable.
For Example: 
  p for Product
  sod or detail or sales for SalesOrderDetail etc.